modules/libcom/src/osi/compiler/gcc/compilerSpecific.h

```diff
@@ -45,7 +45,11 @@
 /*
  * Enable format-string checking if possible
  */
+#ifdef __USE_MINGW_ANSI_STDIO
```

Comment:
I think the correct test would be `#if __USE_MINGW_ANSI_STDIO` to handle an explicit disable `gcc -D__USE_MINGW_ANSI_STDIO=0`.

```
$ grep -R __USE_MINGW_ANSI_STDIO /usr/x86_64-w64-mingw32/include/
/usr/x86_64-w64-mingw32/include/swprintf.inl:#if __USE_MINGW_ANSI_STDIO
/usr/x86_64-w64-mingw32/include/swprintf.inl:#if __USE_MINGW_ANSI_STDIO
...
```